values-local.yaml

```diff
@@ -4,10 +4,20 @@ kiali:
       # in local deployment we have insecure staging
       # let's encrypt certs
       insecureSkipVerifyTls: true
-  resources: ~
+  podAnnotations: ~
+  # we have to set these values for local deployment
+  # if null kiali-operator uses it's defaults
```

Comment:
if null kiali-operator uses it's defaults
=>
 if null, kiali-operator uses its defaults